doc/md/cli/reference.md

```diff
@@ -145,13 +145,15 @@ atlas migrate validate [flags]
 
 #### Details
 'atlas migrate validate' computes the integrity hash sum of the migration directory and compares it to 
-the atlas.sum file. If there is a mismatch it will be reported.
+the atlas.sum file. If there is a mismatch it will be reported. If the --dev-url flag is given, the migration files are 
+executed on the connected database in order to validate SQL semantics.
```

Comment:
Additional option to validate can be ensuring migration directory state matches the schema file(s) state, but it should not run by default

Reply:
Definetly, yes!